tools/editliveos

```diff
@@ -2346,8 +2342,8 @@ def main():
             editor._configure_bootloader(editor._LiveImageCreatorBase__isodir)
         if editor.ks:
             # Run_pre_scripts same code as ImageCreator_run_post_scripts
-            editor._run_post_scripts()
-            editor.install_rpms()
+            # editor._run_post_scripts()
+            editor.install()
```

Comment:
Where is this function coming from?

Reply:
this is inherited from the ```ImageCreator``` class, in the ```creator.py``` file. The chain of inheritance is LiveImageEditor->LiveImageCreator->LoopImageCreator->ImageCreator.